src/coreclr/src/vm/eeconfig.cpp

```diff
@@ -723,6 +724,20 @@ fTrackDynamicMethodDebugInfo = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_
     if (IsCompilationProcess())
         iGCconcurrent = FALSE;
 
+    bool gcLarge = Configuration::GetKnobBooleanValue(W("System.GC.LargePages"), false);
```

Comment:
There is `CLRConfig::EXTERNAL_GCLargePages` that takes `CLRConfig::EXTERNAL...`.

Can this be just `Configuration::GetKnobBooleanValue(W("System.GC.LargePages"), CLRConfig::EXTERNAL_GCLargePages)` ?

Reply:
Did you try this overload? Or is there a reason why it does not work?